src/components/WebexMember/WebexMember.scss

```diff
@@ -24,7 +29,7 @@
   .presenter {
     @extend .subtext;
   }
-
+  
```

Comment:
Careful of unnecessary whitespace. Please remove

Reply:
Done.